build.yaml

```diff
@@ -4,6 +4,7 @@ targets:
       mockito|mockBuilder:
         generate_for:
           - test/end2end/*.dart
+          - example/**.dart
```

Comment:
Should I change here? If not, I will remove it.

Reply:
No that's great. But let's go ahead and alphabetize it, by putting it one line up.